src/libraries/System.Runtime.Serialization.Schema/tests/System.Runtime.Serialization.Schema.Tests.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
nit: please follow the same code styling convention as in other projects that you are touching in this PR by separating the PropertyGroups and ItemGroups with an empty line.